<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Index_basicCh</class>
 <widget class="QDialog" name="Index_basicCh">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res_Index.qrc">
    <normaloff>:/icons/icons/food/38.png</normaloff>:/icons/icons/food/38.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="3,4" columnminimumwidth="4,3">
   <property name="leftMargin">
    <number>40</number>
   </property>
   <property name="rightMargin">
    <number>40</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;姓名&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nameEdit">
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>5</number>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;年龄&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="ageEdit">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;体重&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="weightEdit">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;身高&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="heightEdit">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>账号名</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="IDEdit">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QRadioButton" name="maleRadio">
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>男(male)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QRadioButton" name="femaleRadio">
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>女(female)</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="OKButton">
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="text">
      <string>确认修改</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res_Index.qrc"/>
 </resources>
 <connections/>
</ui>
